config/initializers/inflections.rb

```diff
@@ -14,3 +14,8 @@
 # ActiveSupport::Inflector.inflections(:en) do |inflect|
 #   inflect.acronym 'RESTful'
 # end
+
+ActiveSupport::Inflector.inflections(:en) do |inflect|
+  inflect.acronym 'DOI'
```

Comment:
Required when `.humanize` is called on doi, it will become `DOI` instead of `Doi`